app/models/org.rb

```diff
@@ -0,0 +1,20 @@
+class Org < ApplicationRecord
+  validates :handle, presence: true,
+    uniqueness: { case_sensitive: false },
+    length: { within: 2..40 },
+    format: { with: /\A[A-Za-z][A-Za-z_\-0-9]*\z/ }
```

Comment:
should we disallow ending on a `[_-]`?

Reply:
I don't think so, unless we also change the User handle rules, since we need them to be consistent.

Reply:
in that case, let's extract the pattern to a constant under `Patterns` and use that for both